### PR TITLE
Update apn configuration docs

### PR DESCRIPTION
**Pull request by avast1268**
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/lte/deploy_config_apn.md [hunk 1]
```diff
@@ -19,41 +21,122 @@ first APN in the subscriber profile as the default APN and establishes a
...
 subscriber profiles and any new attaches as well as PDN connection requests
 will be impacted by these changes.
 
-- You can also add a new APN configuration by clicking on the "Add APN"
+![APN tab in NMS](assets/nms/APN/Configuration_By_NMS.png)
+
+
+
+
+
```
Comment (hcgatewood) [inline docs/readmes/lte/deploy_config_apn.md:43]: No need for these extra newlines

File: docs/readmes/lte/deploy_config_apn.md [hunk 2]
```diff
@@ -19,41 +21,122 @@ first APN in the subscriber profile as the default APN and establishes a
...
+- You can also add a new APN configuration by clicking on the "Create New APN"
 button and filling out the requested fields. After saving these changes, the
 page should refresh with the new list of APNs and their configurations.
 
-![Creating an APN Configuration](assets/nms/add_apnconfig.png)
+![Defining new APN](assets/nms/APN/define_new_APN.png)
+
+  - APN ID: Specifies the unique name used for this APN which could be configured
+  on different terminals
+
```
Comment (hcgatewood) [inline docs/readmes/lte/deploy_config_apn.md:53]: Nit: no need for these newlines between list points

File: docs/readmes/lte/deploy_config_apn.md [hunk 3]
```diff
@@ -19,41 +21,122 @@ first APN in the subscriber profile as the default APN and establishes a
...
+  indicating that resources of a service can be preempted by a service with higher
+  Priority Level value.
+
 
 ## Add APN Configurations to Subscriber Profiles
 
 The next step is to add one or more APN configurations to subscriber profiles
-so that UEs can start consuming network services based on their APNs:
+so that UEs can start consuming network services based on their APNs. From the left pane
+choose the Sunscriber tab.
```
Comment (hcgatewood) [inline docs/readmes/lte/deploy_config_apn.md:78]: Typo (?)

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Hey Ava, thanks for adding this PR!

One note on the content: looks like you're adding some png files that aren't being referenced in the actual documentation. Let's go ahead and remove the ones that are unused

And for passing CI: the DCO check and semantic pr check should provide comments on how to fix them. You'll also want to follow our [Docs Precommit](https://magma.github.io/magma/docs/next/docs/docs_overview#precommit) howto for linting your doc changes

**Comment by ekowtaylor** [comment]
@avast1268 what support is needed to merge these changes?

**Comment by ekowtaylor** [comment]
@avast1268 any interests in moving forward with this PR?

**Comment by ekowtaylor** [comment]
Lets reopen and rebase if you plan to move forward with this.

**Pull request closed without merging**